subprojects/docs/src/docs/userguide/signingPlugin.adoc

```diff
@@ -195,16 +197,24 @@ For a task to be “signable”, it must produce an archive of some type. Tasks
 [[sec:conditional_signing]]
 ==== Conditional Signing
 
-A common usage pattern is to only sign build artifacts under certain conditions. For example, you may not wish to sign artifacts for non-release versions. To achieve this, you can specify that signing is only required under certain conditions.
+A common usage pattern is to only require signing build artifacts under certain conditions. For example, you may not need to sign artifacts for non-release versions. To achieve this, you can specify that signing is only required under certain conditions.
 
 ++++
-<sample id="conditionalSigning" dir="signing/conditional" title="Conditional signing">
-                <sourcefile file="build.gradle" snippet="conditional-signing"/>
-            </sample>
+<sample id="conditionalSigning" dir="signing/conditional" title="Specifying when signing is required">
+    <sourcefile file="build.gradle" snippet="conditional-signing"/>
+</sample>
 ++++
 
 In this example, we only want to require signing if we are building a release version and we are going to publish it. Because we are inspecting the task graph to determine if we are going to be publishing, we must set the `signing.required` property to a closure to defer the evaluation. See api:org.gradle.plugins.signing.SigningExtension#setRequired(java.lang.Object)[] for more information.
 
+If the `required` condition does not hold true, artifacts will only be signed if signatory credentials are configured. Alternatively, you may want to disable signing entirely whether or not signatory credentials are configured. If so, you can configure when the `Sign` tasks are enabled, e.g. using `onlyIf` like in the following example.
```

Comment:
Consider:

    If so, you can disable the `Sign` tasks, for example by using the `onlyIf()` method shown in the following example:

And can we link to another user manual section that explains how to disable Gradle tasks? I think [this is the link](https://docs.gradle.org/current/userguide/more_about_tasks.html#sec:skipping_tasks) and we should probably use the same terminology of "skipping tasks".